app.js

```diff
@@ -85,6 +85,7 @@ var processRequest = function (req, res, isTar) {
   var token = (!isTar && req.body) ? req.body.token : null;
   var tar = (isTar) ? req.file : null;
   var user = req.user ? req.user : null;
+  var dryRun = Boolean(req.body && req.body['dry-run'] && /^(true|t|yes|y|1|on)$/i.test(req.body['dry-run']));
```

Comment:
I'm not convinced allowing multiple value for dry-run is really useful. I would go with `true` only but no big deal.